code/modules/antagonists/heretic/status_effects/mark_effects.dm

```diff
@@ -260,3 +260,52 @@
 /datum/status_effect/eldritch/knock/on_remove()
 	REMOVE_TRAIT(owner, TRAIT_ALWAYS_NO_ACCESS, STATUS_EFFECT_TRAIT)
 	return ..()
+
+// MARK OF MOON
+
+/datum/status_effect/eldritch/moon
+	effect_icon_state = "emark8"
+	///Used for checking if the pacifism effect should end early
+	var/damage_sustained = 0
+
+/datum/status_effect/eldritch/moon/on_apply()
+	.=..()
+	ADD_TRAIT(owner, TRAIT_PACIFISM, type)
+	owner.emote(pick("giggle", "laugh"))
+	owner.balloon_alert(owner, "you feel unable to hurt a soul!")
+	RegisterSignal (owner, COMSIG_MOB_APPLY_DAMAGE, PROC_REF(on_damaged))
+	return TRUE
+
+/// Signal proc for [COMSIG_MOB_APPLY_DAMAGE], being damaged past a threshold will remove the PACIFISM trait
+/datum/status_effect/eldritch/moon/proc/on_damaged(datum/source, damage, damagetype)
+	SIGNAL_HANDLER
+
+	// The grasp itself deals stamina damage so we will ignore it
+	if(damagetype == STAMINA)
+		return
+
+	// Adds damage to the damage sustained
+	damage_sustained += damage
+
+	// If the damage_sustained is equal or below 15 don't remove the trait
+	if(damage_sustained<15)
+		return
+
+	// Remove the pacifism and give them a balloon alert
+	REMOVE_TRAIT(owner, TRAIT_PACIFISM, type)
+	owner.balloon_alert(owner, "you feel able to once again strike!")
+
+/datum/status_effect/eldritch/moon/on_effect()
+	if(ishuman(owner))
+		var/mob/living/carbon/carbon_owner = owner
```

Comment:
`ishuman`, then casts to carbon? I don't think any of this code is human specific anyways tbh, it should work with just `owner`

Reply:
changed it to work like that